src/browser/nativeModule.ts

```diff
@@ -84,8 +84,8 @@ export const browserNativeModuleRNPurchases = {
   setLogLevel: async (level: string) => {
     PurchasesCommon.setLogLevel(level);
   },
-  setLogHandler: async (_handler: (message: string) => void) => {
-    methodNotSupportedOnWeb('setLogHandler');
+  setLogHandler: async (handler: (message: string) => void) => {
```

Comment:
is this a change in the public API?

Reply:
No, this is on the nativemodule side which is not exposed and just changing the name of a local variable, so I don't think it's a breaking change